src/endpoint/s3/s3_rest.js

```diff
@@ -147,7 +146,20 @@ async function handle_request(req, res) {
     await http_utils.read_and_parse_body(req, options);
     const reply = await op.handler(req, res);
     http_utils.send_reply(req, res, reply, options);
+
     collect_bucket_usage(op, req, res);
+
+    let bucket_info;
```

Comment:
bucket_info is not used outside the following if scope.
it's better to remove this line and add const inside the scope

Reply:
Done